calblueprint/unloop
# AddFromExisting Component

This PR contains code to display the AddFromExistingModal. Search and category filtering should be fully functioning. Styling should also be proper to the figma (save for the actual sizing of the component).

## What DOESNT work
- Edit/Delete buttons on the ActionItemCard I made. Had to make my own as a placeholder for CJs to get this PR in quicker. We can swap this ActionItemCard out for the one CJ makes (which should be fully functional) once his PR is ready to be merged in.

## To test:
- Run app and go to `/assignments` endpoint. Click around and move the viewport to try and break component
- Check adherence to Figma
- Check search and filter functionality works properly and in entirety


## Component in figma looks like:
![image](https://user-images.githubusercontent.com/35501399/79178414-10f37080-7dba-11ea-8293-30d9e47368e8.png)

## Created component looks like: (relative sizes can be changed)
![image](https://user-images.githubusercontent.com/35501399/79178788-01285c00-7dbb-11ea-8ebe-f0fe3c95b855.png)




CC: @didvi
